include/highfive/H5Object.hpp

```diff
@@ -51,12 +79,40 @@ class Object {
     hid_t _hid;
 
   private:
+    // Init with an low-level object id
+    Object(hid_t);
+
     template <typename Derivate>
     friend class NodeTraits;
     template <typename Derivate>
     friend class AnnotateTraits;
 };
-}
+
+
+///
+/// \brief A class for accessing hdf5 objects info
+///
+class ObjectInfo  {
+  public:
+    /// \brief Retrieve the address of the object (within its file)
+    haddr_t getAddress() const noexcept;
```

Comment:
`getAddress` is not consistent with the other member variables that are not prefixed by `get`.
The naming convention in HighFive is in favor of prefixing accessor with `get`, so I would rename the other member functions instead.

Reply:
Yes to consistency! thanks!